code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -178,3 +178,43 @@ var/global/list/blood_list = list()
 	random_icon_states = list("mucus")
 
 	var/dry=0
+
+/obj/effect/decal/cleanable/blood/gibs/blood
+	name = "blood"
+	icon = 'icons/effects/blood.dmi'
+	random_icon_states = list("mfloor1", "mfloor2", "mfloor3", "mfloor4", "mfloor5", "mfloor6", "mfloor7")
+	amount = 5
+	base_icon = 'icons/effects/blood.dmi'
+
+/obj/effect/decal/cleanable/blood/gibs/blood/update_icon()
```

Comment:
Is all this just copypaste from the blood cleanable decal?
That's really bad. Can't you just spawn the decals instead of this thing?